parser.makam

```diff
@@ -77,14 +87,18 @@ syntax.rule goident_ (charcons ident_first (charmany ident_rest)).
     (fun x => fun y => (tuple (concrete.name exprvar x) y)) { <goident> <goident> }
 
   expr ->
-    method_call { <baseexpr> "." <goident> "(" <list_sep (token ",") expr> ")" }
-  / struct_lit  { <goident> "{" <list_sep (token ",") expr> "}" }
-  / select      { <baseexpr> "." <goident> }
-  / type_ass    { <baseexpr> "." "(" <goident> ")" }
+    expr_action { <baseexpr> "." <list_sep_plus (token ".") action > }
```

Comment:
What is list_sep_plus? What's the difference with list_sep?

Reply:
At least one element.

https://github.com/astampoulis/makam/blob/a6c68478c57a09fd549af920ec9ae6ba39399a6a/stdlib/syntax/library.makam#L21